examples/jsm/postprocessing/BokehPass.js

```diff
@@ -33,7 +34,8 @@ class BokehPass extends Pass {
 
 		this.renderTargetDepth = new WebGLRenderTarget( 1, 1, { // will be resized later
 			minFilter: NearestFilter,
-			magFilter: NearestFilter
+			magFilter: NearestFilter,
+			type: HalfFloatType
```

Comment:
@Mugen87 Can you please explain why `NearestFilter` is used in many of the examples in this PR? Should it not be a fallback, instead?

Reply:
I did not want to touch filtering settings since they might be intended for specific passes. 

However, I'm open for a review in this context and update to a higher setting when it's appropriate.